documentation/feature/rangecard.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: wiki
+title: Rangecard
+description: Add a range card for your weapons
+group: feature
+parent: wiki
+---
+
+## 1. Overview
+
+Add a range card that update itself with your weapon and the type of ammo you're using.
+
+## 2. Usage
+
+### 2.1 Opening the range card
+- Open the self interaction menu <kbd>CTRL</kbd> + <kbd>left windows</kbd>
+- Select `equipment`
+- Select `Open Range Card`
+
+### 2.2 Using the range card
+- To use this to it's full potential the use of a `vector 21` is strongly recommended, a `kestrel 4500` will also help.
+
+- Pull out your trusty rangefinder (`Vector 21` preferred) and get the distance between you and your target.
+
+- Open your rangetable and look under the `Target range` column.
+
+- Move to the `Bullet Drop` column, the drop is in MRADs, you need to compensate for it by adjusting your sight. Example, you want to adjust for a bullet drop of -7.9 MRADs  simply adjust your scope 7.9 MRADs vertically. (check [feature scopes](http://ace3mod.com/wiki/feature/scopes.html) ) for this.
```

Comment:
`Target range` or `Target Range`
`Bullet Drop` or `Bullet drop`

should be the same